hardhat-scripts/deploy-strategy.ts

```
@@ -0,0 +1,130 @@
+
+async function deployEventEmitter(managedSigner: NonceManager) {
+ const admin = process.env.ADMIN_ADDRESS!;
+ const devops = process.env.DEVOPS_ADDRESS!;
+
+ const EventEmitter = (await ethers.getContractFactory("TermVaultEventEmitter")).connect(managedSigner);
+ const eventEmitterImpl = await EventEmitter.deploy();
+ await eventEmitterImpl.deployed();
+ 
+ console.log("Deployed event emitter impl to:", eventEmitterImpl.address);
+
+ const Proxy = (await ethers.getContractFactory("ERC1967Proxy")).connect(managedSigner);
```

Biome (1.9.4)

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)